packages/@uppy/core/types/index.d.ts

```diff
@@ -318,7 +320,7 @@ export class Uppy {
     file: AddFileOptions<TMeta>
   ): string
 
-  removeFile(fileID: string): void
+  removeFile(fileID: string, reason: FileRemoveReason): void
```

Comment:
The argument is optional: https://github.com/transloadit/uppy/blob/56a67de550c5d774250b476f72a5b26310b32ec5/packages/@uppy/core/src/Uppy.js#L906
```suggestion
  removeFile(fileID: string, reason?: FileRemoveReason): void
```

Reply:
You are right, I forgot that. Thanks.